packages/scripts/config/webpack.config.js

```diff
@@ -184,9 +184,15 @@ const config = {
 			},
 			{
 				test: /\.svg$/,
+				issuer: /\.jsx?$/,
 				use: [ '@svgr/webpack', 'url-loader' ],
 				type: 'javascript/auto',
 			},
+			{
+				test: /\.svg$/,
+				issuer: /\.(sc|sa|c)ss$/,
+				type: 'asset/inline',
```

Comment:
In `wp-scripts@17` the svg files was inlined so `asset/inline` is used there, but should this be changed to `asset/resource` and move the svgs to the `images` folder?

Reply:
That's interesting. So we can have now different handling in CSS and JS, awesome.

We should check how it worked with with `@wordpress/scripts` v17 that was using webpack 4 and provide the same experience for backward compatibility.

Reply:
https://unpkg.com/browse/@wordpress/scripts@17.0.0/config/webpack.config.js

It was as simple as:

```js
{
    test: /\.svg$/,
    use: [ '@svgr/webpack', 'url-loader' ],
},
```

Reply:
Do you think we can add `svg` to the resource handler for other images like gid or png?

Reply:
> Do you think we can add `svg` to the resource handler for other images like gif or png so they have consistent processing?

Yes, we can do that. That will create a svg file in the `images` folder and not inline them like `url-loader` did in v17. However I think that is fine, and personally I would prefer it. Both approaches has pros and cons.

What do you think @gziolo. Inline them or move svgs to the resouce handler?

Reply:
I see now that the default was no limit, so it was inlining all SVG files. Processing all the changes in webpack 5 API is a mountain of work. 

Let me test the current setup with some custom plugin and I will report back 😄

Reply:
We could use the general `asset` type, where webpack will automatically choose between `resource` and `inline` based on filesize.

See https://webpack.js.org/guides/asset-modules/#general-asset-type

Reply:
This page is helpful. Let's try back compact version first. We can iterate further 😄 

On the bright side, the refactoring they applied makes a ton of sense ❤️